src/main/io/serial_cli.c

```diff
@@ -547,6 +547,10 @@ static const char * const lookupTablePwmProtocol[] = {
 #endif
 };
 
+static const char * const lookupTablePwmPolarity[] = {
+    "POSITIVE", "NEGATIVE"
+};
```

Comment:
I generally don't like to use booleans for named things, since I always get confused as to which is which.

How about "STANDARD" and "INVERTED"?